frontend/src/Commons/Styles/Button.style.ts

```diff
@@ -11,14 +11,13 @@ const StyledButton = styled.button`
   color: ${tokens.typographyColorPrimaryInvert};
   border-radius: 8px;
   border: 0;
-  padding:10px;
+  padding: 10px;
```

Comment:
If you haven't already you should search for and add the "Prettier - Code formatter" in VSCode.

<img width="1724" alt="Screenshot 2024-06-05 at 3 57 27 PM" src="https://github.com/bcgov/nr-sustainment-capstone-2024/assets/20193291/f09be4eb-d228-4579-99e4-9d4706d62d60">

Then click Code > Settings... > Settings and search for "Default Formatter" set this to Prettier and then search for and check "Format on Save".

<img width="807" alt="Screenshot 2024-06-05 at 4 00 23 PM" src="https://github.com/bcgov/nr-sustainment-capstone-2024/assets/20193291/a4952c40-9d35-4a40-aa02-617f8cfc1c9e">
<img width="790" alt="Screenshot 2024-06-05 at 4 01 09 PM" src="https://github.com/bcgov/nr-sustainment-capstone-2024/assets/20193291/290c710d-0397-456c-89a5-9a778a8cd2bf">

You can also choose whether you format your modifications or the whole file.

Reply:
We've implemented Husky, so it will run linter and formatter before it gets pushed. It should be fix by now. Thanks for the comment!

Reply:
Worth noting that the GitHub syntax highlighting might be a little misleading
`Emotion.js` the tool they're using for Styled components, **uses string interpolation** so this will slip past linters undetected because its a string. 

Being said, I do enjoy using Emotion a lot, There is some `some` support for object notation instead of the string interpolation but I haven't played with it myself to know if its limited or as fully fleshed out (I always used the interpolated version).

A grammar/spellcheck extension might help in catching these cases :) 
There is always object styles. But I havent used this, it may have limitations
https://emotion.sh/docs/object-styles

Reply:
But if you don't have prettier yet, Fergmac is 100% correct. Catching it early prevents later frustruation. The Addition of Husky also helps as an additional line of defense, good on your team for researching/adding that one :)

Reply:
Kudos to Gui for implementing it. 🥳